pkgs/native_toolchain_c/test/cbuilder/clinker_test.dart

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2024, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:native_assets_cli/native_assets_cli.dart';
+import 'package:native_toolchain_c/native_toolchain_c.dart';
+import 'package:test/test.dart';
+
+import '../helpers.dart';
+
+void main() {
```

Comment:
I saw this on other files - what does it mean?

Reply:
```
@TestOn('windows')
@OnPlatform({
  'windows': Timeout.factor(10),
})
library;
```

Is not respected by the test runner inside the Dart SDK. So in order to prevent those tests from failing in the Dart SDK CI, we just return not having executed anything.